_opensearch/install/plugins.md

```diff
@@ -267,6 +273,7 @@ Members of the OpenSearch community have built countless plugins for the service
 - [Index State Management]({{site.url}}{{site.baseurl}}/im-plugin/ism/index/)
 - [k-NN]({{site.url}}{{site.baseurl}}/search-plugins/knn/index/)
 - [ML Commons Plugin]({{site.url}}{{site.baseurl}}/ml-commons-plugin/index/)
+- [Neural Search]({{site.url}}{{site.baseurl}}/neural-search/index/)
```

Comment:
Neural search should be a search plugin

Reply:
I see. Just want to confirm that this plugin will be found under Search plugins folder?